extensions/mssql/src/objectManagement/commands.ts

```diff
@@ -414,6 +414,7 @@ async function handleDropDatabase(context: azdata.ObjectExplorerContext, service
 }
 
 function getDialog(service: IObjectManagementService, dialogOptions: ObjectManagementDialogOptions): ObjectManagementDialogBase<ObjectManagement.SqlObject, ObjectManagement.ObjectViewInfo<ObjectManagement.SqlObject>> {
+	const dialogWidth = '750px';
```

Comment:
Could you make this name a bit more descriptive since it's in the shared commands file?